src/chttpd/src/chttpd.erl

```diff
@@ -537,15 +641,22 @@ extract_cookie(#httpd{mochi_req = MochiReq}) ->
 %%% end hack
 
 set_auth_handlers() ->
-    AuthenticationDefault =  "{chttpd_auth, cookie_authentication_handler},
-      {chttpd_auth, default_authentication_handler}",
+    AuthenticationDefault =
+        "{chttpd_auth, cookie_authentication_handler},\n"
```

Comment:
0

Reply:
0

I think it auto-inserted a `\n` character in there and split it into two lines. The result is the same  but would need a bit extra scrutiny since it is the authentication handler of all places. We may want by bypass these sections even on the first go around.

Reply:
Done in 2de42ba664da9c0e7d74ef7172489457e7c3f258

Reply:
-1